src/main/java/com/jalasoft/webservice/model/ZipFiles.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.jalasoft.webservice.model;
+
+import com.jalasoft.webservice.utils.Utils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+/**
+ * The class compress files.
+ *
+ * @author Fernando Hinojosa on 9/30/19.
+ * @version v1.0
+ */
+public class ZipFiles {
+
+    /**
+     * Creates a Zipfile with a list of files.
+     *
+     * @param filePaths
+     */
+    public void zipFiles(String [] filePaths) {
+        try {
+            Utils util = new Utils();
+            File firstFile = new File(filePaths[0]);
+            String zipFileName = util.getPublic() + firstFile.getName().concat(".zip");
+
+            FileOutputStream fos = new FileOutputStream(zipFileName);
+            ZipOutputStream zos = new ZipOutputStream(fos);
+
+            for (String aFile : filePaths) {
+                zos.putNextEntry(new ZipEntry(new File(aFile).getName()));
+
+                byte[] bytes = Files.readAllBytes(Paths.get(aFile));
+                zos.write(bytes, 0, bytes.length);
+                zos.closeEntry();
+            }
+
+            zos.close();
+
+        } catch (FileNotFoundException ex) {
+            System.err.println("A file does not exist: " + ex);
```

Comment:
In our web service we don´t need to use println.

Reply:
Solved